package.json

```diff
@@ -36,8 +36,13 @@
     },
     "dependencies": {
         "bignumber.js": "^9.0.0",
-        "ethers": "^4.0.39",
         "isomorphic-fetch": "^2.2.1",
         "typescript": "^3.8.3"
+    },
+    "peerDependencies": {
```

Comment:
The script `build` will fail for me with the message
```
error TS2307: Cannot find module '@ethersproject/abi'.
```
If the ethers dependencies are not in `dependencies`. Do you also have the same error?

Reply:
Ah sorry, I forgot to add it as a dev dependency. It should be able to be built while using the user's installation of ethers now.